docs/install/Anthos.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Anthos"
+linkTitle: "Anthos"
+weight: 20
+---
+
+## Overview
+
+This guide will walk you through installing Ondat onto an [Anthos cluster](https://cloud.google.com/anthos) using the [Ondat kubectl plugin](https://docs.ondat.io/docs/reference/kubectl-plugin/).
+
+## Prerequisites
+
+* You have met the minimum resource requirements for Ondat to successfully run. Review the main [Ondat prerequisites](https://docs.ondat.io/docs/prerequisites/) page for more information.
+
+* The following CLI utilities are installed on your local machine and are available in your `$PATH`:
+>- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) 
+>- [kubectl-storageos](https://docs.ondat.io/docs/reference/kubectl-plugin/) 
+
+* You have a running Anthos user cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
+
+* Your Anthos clusters use [ubuntu_containerd](https://cloud.google.com/anthos/clusters/docs/on-prem/1.8/concepts/using-containerd) as the default node operating system. This version was chosen as the image includes the native Linux components required for Ondat.
+
+## Procedure
+
+### Step 1 - Conducting Preflight Checks
+
+- Run the following command to conduct preflight checks against the Anthos cluster to validate that Ondat prerequisites have been met before attempting an installation.
+
+```bash
+kubectl storageos preflight
+```
+
+### Step 2 - Installing Ondat
+
+1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance.
+
+```bash
+export STORAGEOS_USERNAME="admin"
+export STORAGEOS_PASSWORD="password"
+``` 
+
+2. Run the following  `kubectl-storageos` plugin command to install Ondat.
+
+```bash
+kubectl storageos install \
+  --include-etcd \
```

Comment:
What `StorageClass` is used here by default, `local-path` or the default cluster one?

I'm thinking if it's `local-path` we'll want to stipulate non-production-readiness or suggest adjusting it.

Reply:
Hi Callum,

This was an Anthos on vSphere install so this provides:

kubernetes.io/vsphere-volume 

This is the standard and what I used for etcd.

For the production-readiness we link to the master statement from this:

You have met the minimum resource requirements for Ondat to successfully run. Review the main [Ondat prerequisites](https://docs.ondat.io/docs/prerequisites/) page for more information.

This seems better than maintaining the production statements in every single page?

Chris